Cpp/source/DrawingEngine/DrawingEngine.cpp

```diff
@@ -0,0 +1,31 @@
+#include "DrawingEngine.hpp"
+
+DrawingEngine::DrawingEngine(int buf_size)
+{
+    buffer_size = buf_size;
+    FrameBuffer = new Pixel*[buffer_size];
+
+    for(int i=0; i<buffer_size; i++)
+    {
+        FrameBuffer[i] = new Pixel[buffer_size];
+    }
+
+    for(int i=0; i<buffer_size; i++)
```

Comment:
You should fill it with zeros, best using some other method than nested for loops

Reply:
done